provider/datagen/tests/test-options.rs

```diff
@@ -210,29 +264,27 @@ fn test_fallback_options() {
         .unwrap();
     let data_explicit_hybrid = testing_exporter.take_map_and_reset();
 
-    // Explicit locales are "arc", "ar-EG", "en-GB", "es", "sr-ME", "ru-Cyrl-RU"
+    // Explicit locales are "arc", "ar-EG", "en-GB", "sr-ME"
     let explicit_hybrid_locales = [
         "ar",              // ancestor of ar-EG
         "ar-EG",           // explicit locale
         "ar-EG-u-nu-latn", // descendant of ar-EG
         // "ar-u-nu-latn", // ??? should this be included?
-        "arc",    // Aramaic, inheriting from und
-        "en",     // ancestor of en-GB
-        "en-001", // ancestor of en-GB
-        "en-GB",  // explicit locale not in supported locales
```

Comment:
Issue: the en-001 vs en-GB being included due to different reasons (one in supported locales, the other in explicit locales) is interesting behavior worth testing.